### ISSUE TITLE
Same pain voice is played even though player selects a man voice

### ISSUE THREAD

**Issue by yomox9** [nmrih/source-game#866]
All men's pain voice is same.
By the way, There are pugman's pain voices in nmrih_dir.vpk
Version 1.09.7

**Comment by Dysphie** [comment]
Pugman's bleeding grunts are waaaay too long to be used. _And uncomfortable._

**Comment by MintyDysphie** [comment]
Pugman's bleeding grunts are waaaay too long to be used. _And uncomfortable._

**Comment by yomox9** [comment]
Yes, I think It's not a bug. But I thought that we should discuss this.

**Comment by Fersigal** [comment]
What is this?
Did you read following?
https://github.com/ZPSNMRiHIssues/nmrihissues

**Comment by yomox9** [comment]
Yes. I already read it before to post. This is my servers players question. I thought that same voice is strange.

**Comment by AtheistNMRiH** [comment]
there are just no sound files for other voices, not sure if they can be recorded now
as for pugman, iirc that was fixed

**Comment by felis-catus** [comment]
Pugman pain & bleeding voices have been added (and adjusted) in 1.10. These lines didn't exist in other male sets, but the voice system now has support for changing pain sounds.

**Comment by Bubka3** [comment]
^ above and voice sets w/o pain and bleeding voices will default to default ones
